test/modules/engin/pkg_module_res.future

```diff
@@ -0,0 +1,6 @@
+bug: function resolution on package modules
+
+Lydia's comment on the issue:
```

Comment:
Sorry, pronoun trouble.  I meant that for all cases that a standard module's explicit naming works, the package module's explicit naming doesn't.